src/app-layer-htp-range.c

```diff
@@ -132,10 +132,8 @@ static inline bool ContainerValueRangeTimeout(void *data, const SCTime_t ts)
     HttpRangeContainerFile *cu = data;
     // we only timeout if we have no flow referencing us
     if ((uint32_t)SCTIME_SECS(ts) > cu->expire || cu->error) {
-        if (SC_ATOMIC_GET(cu->hdata->use_cnt) == 0) {
```

Comment:
Are you sure to remove this condition ? cf comment above

Reply:
is handled in `THashExpire` now

Reply:
Nice so the comment above should go away as well, right ?